pillar/vector/init.sls

```diff
@@ -0,0 +1,2 @@
+vector:
+  configuration:
```

Comment:
There's not really a need for an `init.sls` file if it's not doing anything. The `init.sls` is just for providing a default baseline that gets merged into specific configs for different instances.

Reply:
ok. I'll remove this.

Reply:
Removed in 34f969e1